assets/excuses.txt

```diff
@@ -0,0 +1,6 @@
+a zombie|the scholarly troll|one of the guardians|a mutated rat|the inkeeper|the drunk executionner|an ugly minstrel|the goblin maid|an elit ork|the trainee wizard|a suspect guy|the barbed prisonner|the herborist|a neighbour's dog|one of the city's guardsmen|a hawker|an adventurer|the plumber|the goblin engineer|a crazy old man
+slipped|skid|broke a thing|shattered a thingy|vomited|lost his keys|did his business|was stuck|got lost|fell|fell asleep|spent the night|woke up|killed himself|hurt himself|stumbled|was stuck|fought|caused problems|put fire
+the cave|the north tunnel|the attic|my office|the ingredient shed|the kitchens|the dogs' kennel|the crows' aviary|the scorpio pit|your office|the second level's stairs|the slime's tray|the main hallway|the workhourse|the forge|the whipping room|the orks' dormitory|the Golbargh's lair|the shop|your bookshop
+this silly|this stupid|a big|an odious|a strange|an incredible|the impossible|the famous|this foolish|the ridiculous|- that's unfortunate - the|the existence of a|the embush caused by a|the trap represented by a|the presence of a|- you're gonna laugh - a|- that's really a shame - the|the position of a|his tendency for a|a mediocre
+rusty wheelbarrow|disformed handle|damaged saw|forgotten basin|wrench|clothing basket|twisted halberd|stuffed owl|trapped terrine|poisoned flute|ghoul head|venomous scythe|disloqued guitar|twisted connecting rod|broken salt shaker|banana peel|decrepit wig|red sock|green slipper|worn slipper
+came from my grandmother|was just there|appeared as if by magic|came from your cousin|had been abandoned|was suspect|shouldn't have been there|truly looke duplicitous|was lurking in the shadows|didn't look like anything|had been left by your neighbour|was well and truly dangerous|was but costy|had a worrying smell|had moved|should've been neat|vous appartient|was in fact slippery|was maybe mine|always causes issues
```

Comment:
What is this?

Reply:
excuse list, see the file name dear

Reply:
excuses, see the file name dear

Reply:
Why is it spread out through 6 lines tho? Since you're using `|` as separator either use that everywhere, not newline, or use only new lines.

Personally, I'd go with new line for each excuse.